.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Try and not commit files like this (as strictly it isn't relevant to the project itself)

Reply:
I'll try to locate where that is as i didn't put it in, thanks for spotting it for me